### ISSUE TITLE
function sumOfTwoLowest(arr)

### ISSUE THREAD

**Issue by gliechty** [den-wdi-2/schedule#3]
Having some real issues with finding the two lowest values of an array and getting their sum (for codewars). I think it has to do with the .splice method I'm using. Checked for answers here: http://stackoverflow.com/questions/5767325/how-to-remove-a-particular-element-from-an-array-in-javascript

and here: http://stackoverflow.com/questions/30427857/finding-the-second-smallest-integer-in-array.

Basically followed the same logical flow from the Stack overflow post, and yet I'm getting really weird results in the console.

Talked to Kevin, he said he wasn't sure.
